demo/app/android/app/src/androidTest/kotlin/integration_test.kt

```diff
@@ -137,13 +138,48 @@ class IntegrationTest {
         println("requestURI ->")
         println( requestURI)
 
-        val requiredOpenID4CIArgs = IssuerInitiatedInteractionArgs(requestURI, crypto, didResolver)
+        val requiredOpenID4CIArgs = InteractionArgs(requestURI, crypto, didResolver)
```

Comment:
`InteractionArgs` and `Interaction`are deprecated - `IssuerInitiatedInteractionArgs` and `IssuerInteraction` should continue to be used. Is there a reason you switched these back?